modules/apache-airflow/templates/values.yaml

```diff
@@ -677,18 +677,17 @@ workers:
     #  limits:
     #   cpu: 100m
     #   memory: 128Mi
-    #  requests:
-    #   cpu: 100m
-    #   memory: 128Mi
+      requests:
```

Comment:
Empty `{}` for resources needs to be deleted

Reply:
Where is the `{}`?

Reply:
NVM!